insights/parsers/tests/test_irqbalance_conf.py

```diff
@@ -0,0 +1,41 @@
+from insights.parsers import irqbalance_conf
+from insights.tests import context_wrap
+from insights.util import keys_in
+
+IRQBALANCE_SYSCONF_TEST = """
+# irqbalance is a daemon process that distributes interrupts across
+# CPUS on SMP systems. The default is to rebalance once every 10
+# seconds. This is the environment file that is specified to systemd via the
+# EnvironmentFile key in the service unit file (or via whatever method the init
+# system you're using has.
+#
+# ONESHOT=yes
+# after starting, wait for a minute, then look at the interrupt
+# load and balance it once; after balancing exit and do not change
+# it again.
+#IRQBALANCE_ONESHOT=yes
+
+#
+# IRQBALANCE_BANNED_CPUS
+# 64 bit bitmask which allows you to indicate which cpu's should
+# be skipped when reblancing irqs. Cpu numbers which have their
+# corresponding bits set to one in this mask will not have any
+# irq's assigned to them on rebalance
+#
+IRQBALANCE_BANNED_CPUS=f8
+
+#
+# IRQBALANCE_ARGS
+# append any args here to the irqbalance daemon as documented in the man page
+#
+IRQBALANCE_ARGS="-d"
+""".strip()
+
+
+def test_irqbalance_conf():
+    ret = irqbalance_conf.SysconfigIrqbalance(context_wrap(IRQBALANCE_SYSCONF_TEST))
+    assert keys_in(["IRQBALANCE_BANNED_CPUS", "IRQBALANCE_ARGS"], ret.data)
```

Comment:
Since its a mixin class with LegacyItemAccess, better remove the ".data" from the test.